src/basic/include/features.h

```diff
@@ -0,0 +1,8 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+#pragma once
+
+#include_next <features.h>
+
+/* Most libc headers includes features.h.
+ * Let's define assert_cc() here, to make it usable in our libc header wrappers. */
+#include <_sd_assert_cc.h>
```

Comment:
I think this commit is overly complicated for the small benefit it brings, can't we just use _Static_assert() directly in the libc header wrappers?

Reply:
OK. Dropped the commit, and now _Static_assert() is used.